### PR TITLE
Add Pantheon.io

### PR DESCRIPTION
**Pull request by ari-gold**

### CODE REVIEW

File: _data/hosting_providers.json [hunk 1]
```diff
@@ -1,4 +1,14 @@
 [
+    {
```
Comment (ohemorange) [inline _data/hosting_providers.json]: *extremely* small nit, but the spacing of this open brace is off by one.
```suggestion
   {
```

File: _data/hosting_providers.json [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+    "name": "Pantheon",
+    "link": "https://pantheon.io/",
+    "category": "full",
+    "tutorial": "https://pantheon.io/docs/guides/launch/",
+    "announcement": "https://pantheon.io/blog/pantheon-launches-global-cdn-automated-https-all-sites",
+    "plan": "",
+    "reviewed": "",
+    "note": "Website Ops for Drupal & WordPress"
```
Comment (ohemorange) [inline _data/hosting_providers.json]: as per the comment above, are there some hosting products you offer that have https set up automatically and others that don't? this is a great field to specify which products offer completely automatic https setup.

File: _data/hosting_providers.json [hunk 3]
```diff
@@ -1,4 +1,14 @@
 [
+    {
+    "name": "Pantheon",
+    "link": "https://pantheon.io/",
+    "category": "full",
+    "tutorial": "https://pantheon.io/docs/guides/launch/",
```
Comment (ohemorange) [inline _data/hosting_providers.json]: just to check -- usually, if users need to go through a tutorial to turn on https for their site, that's what we would call "partial" support. "full" means that if they've set up a site at all, it'll have https. could you help me understand which is the case here?

File: _data/hosting_providers.json (hunk 2 above)
Comment (ari-gold) [reply _data/hosting_providers.json]: Thanks for the review @ohemorange!

 All Pantheon sites include automated HTTPS so I removed the note and tutorial link.

File: _data/hosting_providers.json [hunk 4]
```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "Pantheon",
+    "link": "https://pantheon.io/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "https://pantheon.io/blog/pantheon-launches-global-cdn-automated-https-all-sites",
+    "plan": "",
+    "reviewed": "",
```
Comment (ohemorange) [inline _data/hosting_providers.json]: Ok, looks great so let's mark today as the reviewed date and get this in!
```suggestion
    "reviewed": "2019.6.28",
```

**Pull request merged**